flocker_bb/builders/maint.py

```diff
@@ -75,6 +75,26 @@ def fmap(_f, _value, *args, **kwargs):
     else:
         return _f(_value, *args, **kwargs)
 
+# FLOC-2288
+# New CleanVolumes LoggingBuildStep
+# parameterized on age (lag, like CleanAcceptanceInstances)
+#
+# libcloud configured just as it is in CleanAcceptanceInstances
+#
+# start method which uses libcloud in a thread to list volumes on AWS and Rackspace
+# Flocker-managed Rackspace and AWS volumes have flocker-cluster-id in their metadata
+# Destroy unattached volumes with this tag that are older than the age parameter
+#
+# return result object representing any volumes we destroyed and any we saw and
+# didn't destroy
+#
+# XXX Should really have a way to identify volumes created by the test suite.
+# Maybe also change Flocker test suite so it generates recognizable volumes and
+# then rely on that here.
```

Comment:
For example, it would be nice to run buildbot on flocker, and we don't want its volumes destroyed.